nixos/modules/security/wrappers/default.nix

```diff
@@ -210,5 +247,33 @@ in
             ln --symbolic $wrapperDir ${wrapperDir}
           fi
         '';
+
+    ###### wrappers consistency checks
+    system.extraDependencies = lib.singleton (pkgs.runCommandNoCCLocal
```

Comment:
This seems to be the best way to attach a build time check, inspired by the udev module.

Reply:
apparently runCommandNoCCLocal does not exist anymore:
```
$  nix-instantiate nixos/tests/gnome.nix
error: attribute 'runCommandNoCCLocal' missing, at /home/symphorien/src/nixpkgs/nixos/modules/security/wrappers/default.nix:254:47
(use '--show-trace' to show detailed location information)
```

Reply:
Fixed.